short-response.md

```diff
@@ -0,0 +1,95 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer}`);
+   ```
+    The above code logs an error because the string variable `singer` is declared outside the scope of where the console can log it. 
+
+2. **What happens when we run the following code? Why?**
+   ```javascript
+   const favorite = 'Juan Pablo';
+   console.log(`Our favorite Marcy Lab family member is ${favorite}!`);
+
+   favorite = 'Maya';
+   console.log(`Actually, ${favorite} is my favorite.`);
+   ```
+   The above code logs:
+      "Our favorite Marcy Lab family member is Juan Pablo!" [This is logged because, On line 16 the string variable `favorite` is assigned a value of 'Juan Pablo'. ]
+      "Actually, Maya is my favorite." [This is logged because on line 19 the string variable `favorite` has its value reassigned to 'Maya'.]
+
+3. **What does the following code log? Why?**
+   ```javascript
+   let score = 90; 
+   let newScore = score;
+   score = 100; 
+   console.log(score, newScore);
+   ```
+    The above code logs: "100"
+                         "90"
+
+   This is logged because on line 28, the number variable `score` is assigned a value of 90. 
+   Then on line 29 a new variable named `newScore` is set equal to the number variable `score`. 
```

Comment:
An even better way of saying it:
`newScore` was assigned the value that was referenced by `score`.